src/wireviz/wv_bom.py

```diff
@@ -41,11 +41,24 @@ def get_additional_component_table(
             for part in component.additional_components
             if component.get_qty_multiplier(part.qty_multiplier)
         ]:
-            common_args = {
-                "qty": part.qty * component.get_qty_multiplier(part.qty_multiplier),
-                "unit": part.unit,
-                "bgcolor": part.bgcolor,
-            }
+            
+            if type(part.shorts) == str:
+                numShorts = 1
+            else:
+                numShorts = len(part.shorts)
+            
+            if numShorts > 0:
```

Comment:
This could be avoided, see below.

Reply:
Do not completely understand what you mean

Reply:
@martinrieder ?